consensus/consensus_service.go

```diff
@@ -128,18 +131,23 @@ func (consensus *Consensus) signConsensusMessage(message *msg_pb.Message,
 // UpdateBitmaps update the bitmaps for prepare and commit phase
 func (consensus *Consensus) UpdateBitmaps() {
 	consensus.getLogger().Debug().
-		Str("Phase", consensus.phase.String()).
+		Str("MessageType", consensus.phase.String()).
 		Msg("[UpdateBitmaps] Updating consensus bitmaps")
 	members := consensus.Decider.Participants()
 	prepareBitmap, _ := bls_cosi.NewMask(members, nil)
 	commitBitmap, _ := bls_cosi.NewMask(members, nil)
+	multiSigBitmap, _ := bls_cosi.NewMask(members, nil)
 	consensus.prepareBitmap = prepareBitmap
 	consensus.commitBitmap = commitBitmap
+	consensus.multiSigMutex.Lock()
```

Comment:
Don't want to update existing logic or try to fix it here, just in case it causes new problems. This change is just making sure the new logic is correct.